Allow extra time for TestRangeSplitsWithWritePressure in race mode.

bdarnell opened the pull request:
This test often runs over time, especially with the more verbose
logging added by #436.

spencerkimball commented on kv/split_test.go:
```diff
@@ -169,6 +169,10 @@ func TestRangeSplitsWithWritePressure(t *testing.T) {
 	go startTestWriter(db, int64(0), 1<<15, &wg, nil, nil, done, t)
 
 	// Check that we split 5 times in allotted time.
+	timeout := 3 * time.Second
```
Instead of adding all of this, just make the timeout 10s.

bdarnell replied:
I'll make it 6s (10s would run into issues with the default timeout in `make test`).

spencerkimball commented:
LGTM

bdarnell committed:
Allow extra time for TestRangeSplitsWithWritePressure

This test often runs over time, especially with the more verbose
logging added by #436.
